packages/uniforms/__suites__/renderWithContext.tsx

```diff
@@ -0,0 +1,51 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render as rtlRender } from '@testing-library/react';
```

Comment:
I have a problem with this name, but have no idea for another one... Maybe `renderOnScreen`?